homeassistant/components/switch/wemo.py

```diff
@@ -109,12 +110,33 @@ def device_state_attributes(self):
 
         if self.insight_params or (self.coffeemaker_mode is not None):
             attr[ATTR_CURRENT_STATE_DETAIL] = self.detail_state
+            attr['current_power_w'] = \
```

Comment:
This is just the existing switch attribute divided by 1000.

At some point we'll change the switch attribute to be `W` rather than `mW` anyway.  So this is only really helpful until we do that.

My suggestion would be to remove it